python/open3d/visualization/__init__.py

```diff
@@ -26,11 +26,11 @@
 
 import open3d
 if open3d.__DEVICE_API__ == 'cuda':
-    if "@BUILD_GUI@" == "ON":
```

Comment:
If `@@` configuration is no longer needed, we can remove this file from `cpp/pybind/make_python_package.cmake`.